builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -560,21 +577,35 @@ def buildSlsApp(String runtime, String repo_name) {
 		sh "mkdir library"
 		sh "pip install -r requirements.txt -t library"
```

Comment:
I think this line should go away since we are doing `pip/pip3 install..` below.

Reply:
Removed.